### ISSUE TITLE
User Management Module 

### ISSUE THREAD

**Issue by LjNick** [391DatabaseGroup/RadiologyInformationSystem#7]
This module allows a system administrator to manage (to enter or update) the user information, i.e., the information stored in tables
users,persons, family_doctor.

**Comment by mrgallagher** [comment]
Added Management module. currently persons is the only table it can add to or delete from. edit is partially implemented. as is doctors and users functionality.

**Comment by mrgallagher** [comment]
will do

**Comment by LjNick** [comment]
Matt try throwing a <%@ include file="~/navbar.jsp"

**Comment by mrgallagher** [comment]
that doesnt work. it still looks in the current path but just with a tilda, "/UserManagement/users/~/navbar.jsp" not found

it includes just fine when i use <%@ include file="../../navbar.jsp" %>

the problem im having currently comes when the actual navbar tries to go to a page bec at that instance it is still in the current directory

**Comment by mrgallagher** [comment]
what we need is it to always look in the home directory when navbar.jsp is linking to other files. that way wherever the navbar is included, it still links to the home directory

**Comment by LjNick** [comment]
alright here's my question then, why are you putting the stuff into folders?  According to what documentation I can find the standard way to organize the web app like this is to put all the actual webpages in the same place

**Comment by mrgallagher** [comment]
so you think its best to throw all 15 files ive created into the same location?

**Comment by LjNick** [comment]
Yes when you look at the apache documentation it says that the preferred structure is to have all jsp files in one location: https://tomcat.apache.org/tomcat-7.0-doc/appdev/deployment.html

While it does mention dividing into sub-directories for larger apps ours isn't that big and as we have figured out it introduces other complications so probably easiest to put everything into one folder
